src/aap_eda/core/utils/safe_yaml.py

```diff
@@ -0,0 +1,58 @@
+#  Copyright 2025 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import yaml
+
+
+class UnsafeString(str):
+    def __new__(cls, value):
```

Comment:
What's the purpose of overriding the constructor if we are not doing anything?

Reply:
Tested with this and it works
```
class UnsafeString(str):
    pass
```